example-user-library/README.md

```diff
@@ -13,54 +13,10 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
+# <img src="../logos/logo.svg" width="180">
+## A Tool for Complex and Scalable Data Access Policy Enforcement
 
-# Example Library
+# Example User Library
+Contains the ExampleUser and supporting code.
+The full explanation of how this library is used in context to the example is described in the [Example Library](./example-library/README.md).
```

Comment:
Again pull the user specific things to here rather then being in the example lib